FitGoal/FitGoal/Info.plist

```diff
@@ -37,6 +27,44 @@
 			</array>
 		</dict>
 	</array>
+    <key>LSApplicationQueriesSchemes</key>
+    <array>
+      <string>fbapi</string>
+      <string>fbapi20130214</string>
+      <string>fbapi20130410</string>
+      <string>fbapi20130702</string>
+      <string>fbapi20131010</string>
+      <string>fbapi20131219</string>
+      <string>fbapi20140410</string>
+      <string>fbapi20140116</string>
+      <string>fbapi20150313</string>
+      <string>fbapi20150629</string>
+      <string>fbapi20160328</string>
```

Comment:
Do you need all these values?

Reply:
Reading the last version of the documentation I've found the right version of this code.
According to the documentation: 
This is added to manage Facebook dialogues (e.g., Login, Share, App Invites, etc.) that can perform an app switch to Facebook apps. 
If I run the app without  LSApplicationQueriesSchemes an error happens.